domain/ebl_domain.yaml

```diff
@@ -529,10 +529,6 @@ components:
           properties:
             declaredValue:
               $ref: 'https://api.swaggerhub.com/domains/dcsaorg/DCSA_DOMAIN/1.0.0#/components/schemas/declaredValue'
-        - type: object
-          properties:
-            numberOfRiderPages:
-              $ref: 'https://api.swaggerhub.com/domains/dcsaorg/DCSA_DOMAIN/1.0.0#/components/schemas/numberOfRiderPages'
```

Comment:
Is this a mistake?

Reply:
Oh, yea, forgot to branch. Not a mistake otherwise, but something completely different

Reply:
Will you make a separate PR for removing the numberOfRiderPages? I think that parts needs more discussion ;-) I do agree with your suggestion to update the TransportDocument description

Reply:
Done - sorry for the slobbyness :)